Yoast/wordpress-seo
# Move meta to its own table

## Summary

This PR can be summarized in the following changelog entry:
* Introduces a database table where the link counts are stored.

## Test instructions

This PR can be tested by following these steps:

* The link counts should be saved in a database instead of being calculated on the fly.
* Check if everything still works.
* Check if the counts are saved in the database `{some_prefix}yoast_seo_text_link_counts` after post save and after reindexing.

Fixes #7319
